### DISCUSSION TITLE
How to launch DB ( postgres ) on worker node ?

### DISCUSSION THREAD

**Discussion by fbrbovic** [Dokploy/dokploy#640] [Q&A]
I am using a cluster (swarm)  and when launching a DB using Create Service->Database->Postgres it always launches the DB on Master node.  I would like to launch it on the Worker node.   Is it possible ?

**Comment by Siumauricio** [comment]
Hello, no, it is not possible because when using swarm cluster we do not have access to the remote containers and we can not make backups and everything else so you can not because they have volumes (files), docker swarm works much better for stateless services ie without files, if you want to deploy a database in a remote server and have all the functionality as backups and everything else use multi server feature https://docs.dokploy.com/en/docs/core/multi-server/overview

**Comment by fbrbovic** [comment]
Thank you
